force-app/main/default/classes/SOQL.cls

```diff
@@ -192,8 +184,8 @@ public with sharing class SOQL {
         // Use AggregateQuery to get the total revenue
         AggregateResult[] opportunitySum = [SELECT Sum(Amount) stageSum  FROM Opportunity WHERE StageName = :stage OR Amount > 10000];
         Decimal totalAmount = decimal.Valueof(string.valueof(opportunitySum[0].get('stageSum')));  
```

Comment:
You do have to cast this in order to tell it what format you want to return. However you are doing it twice, first to a string and then to a decimal. You can just cast it directly to a Decimal `(Decimal)` in the front